metrics/prometheus/src/main/java/io/helidon/metrics/prometheus/PrometheusSupport.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
  *
```

Comment:
Can you change this to `2017, 2019"? Thanks.

Reply:
Done.